AISKU/src/Initialization.ts

```diff
@@ -411,3 +427,49 @@ export class Initialization implements IApplicationInsights {
         _this.config.diagnosticLogInterval && _this.config.diagnosticLogInterval > 0 ? _this.config.diagnosticLogInterval : 10000;
     }
 }
+
+// tslint:disable-next-line
+(function () {
+    let sdkSrc = null;
+    let isModule = false;
+
+    try {
+        // Try and determine whether the sdk is being loaded from the CDN
+        // currentScript is only valid during initial processing
+        let scrpt = (document ||{} as any).currentScript;
+        if (scrpt) {
+            sdkSrc = scrpt.src;
+        // } else {
+        //     // We need to update to at least typescript 2.9 for this to work :-(
+        //     // Leaving as a stub for now so after we upgrade this breadcrumb is available
+        //     let meta = import.meta;
```

Comment:
Tried quite a few work-arounds to get this information with 2.5.3 (our current TS version), but nothing worked -- I could have done some sort of post processing (like the es3-poly), but that seems like overkill considering we *should* be upgrading to a later version of TS  sooner rather than later.